test/integration/proportion-boolean/proportion-boolean.test.ts

```diff
@@ -1,3 +1,61 @@
-it('noop', () => {
-  expect(true).toBe(true);
+import { calculate } from '../../../src/calculation/Calculator';
+import { CalculationOptions } from '../../../src/types/Calculator';
+import { getJSONFixture } from '../helpers/testHelpers';
+
+const DEFAULT_CALCULATION_OPTIONS: CalculationOptions = {
+  measurementPeriodStart: '2022-01-01',
+  measurementPeriodEnd: '2022-12-31'
+};
+
+const EARLIER_PERIOD_CALCULATION_OPTIONS: CalculationOptions = {
+  measurementPeriodStart: '2020-01-01',
+  measurementPeriodEnd: '2020-12-31'
+};
+
+const IPP_PATIENT_BUNDLE: fhir4.Bundle = getJSONFixture('proportion-boolean/patients/patient-ipp-bundle.json');
+const DENOM_PATIENT_BUNDLE: fhir4.Bundle = getJSONFixture('proportion-boolean/patients/patient-denom-bundle.json');
+const NUMER_PATIENT_BUNDLE: fhir4.Bundle = getJSONFixture('proportion-boolean/patients/patient-numer-bundle.json');
+const MEASURE_BUNDLE: fhir4.Bundle = getJSONFixture('proportion-boolean/proportion-boolean-bundle.json');
+
+describe('proportion boolean measure', () => {
+  it('calculates ipp patient into ipp', async () => {
+    const results = await calculate(MEASURE_BUNDLE, [IPP_PATIENT_BUNDLE], DEFAULT_CALCULATION_OPTIONS);
+    expect(results.results[0].detailedResults?.[0]?.populationResults).toEqual(
```

Comment:
This assertion is going to be rather annoying to repeat. We can probably create something in `testHelpers` that is a re-usable assertion helper for this kind of thing which will be common in the integration tests. You don't have to take this code exactly, but I was thinking something like:

```ts
function assertPopulationResults(
  result: ExecutionResult<DetailedPopulationGroupResult>,
  expectedPopulations: Partial<Record<PopulationType, boolean>>
) {
  const expectedPopulationResults: PopulationResult[] = Object.entries(expectedPopulations).map(([popType, value]) =>
    expect.objectContaining({
      populationType: popType as PopulationType,
      result: value
    })
  );

  expect(result).toEqual(
    expect.objectContaining({
      detailedResults: expect.arrayContaining([
        expect.objectContaining({
          populationResults: expectedPopulationResults
        })
      ])
    })
  );
}
```

Then in your tests, all you have to do is this:

```ts
assertPopulationResults(results[0], {
  [PopulationType.IPP]: true,
  [PopulationType.NUMER]: false,
  [PopulationType.DENOM]: false
});
```

thoughts?

(side note: typescript enums are terrible, we really should get rid of them lol)